htdocs/langs/de_DE/projects.lang

```diff
@@ -302,3 +302,4 @@ NewLeadbyWeb=Ihre Nachricht bzw. Anfrage wurde erfasst. Wir werden uns so bald w
 NewLeadForm=Neues Kontaktformular
 LeadFromPublicForm=Online-Interessent per öffentlichem Formular
 ExportAccountingReportButtonLabel=Bericht erhalten
+Billable = Verrechenbar
```

Comment:
Update language file is useless. Only en_US must be filled. Other files are erased by transifex sync.

Reply:
Good to know. 
Do I have to revert the change to language files ?

Reply:
Better but not mandatory...